src/Castle.Windsor.Extensions.DependencyInjection/WindsorScopedServiceProvider.cs

```diff
@@ -27,48 +27,40 @@ namespace Castle.Windsor.Extensions.DependencyInjection
 	internal class WindsorScopedServiceProvider : IServiceProvider, ISupportRequiredService, IDisposable
 	{
 		private readonly ExtensionContainerScope scope;
-		private bool disposing = false;
+		private bool disposing;
 
 		private readonly IWindsorContainer container;
 		
 		public WindsorScopedServiceProvider(IWindsorContainer container)
 		{
 			this.container = container;
-			this.scope = ExtensionContainerScope.Current;
+			scope = ExtensionContainerScope.CurrentOrThrow;
 		}
 
 		public object GetService(Type serviceType)
 		{
-			using(var fs = new ExtensionContainerScope.ForcedScope(scope))
+			using(_ = new ForcedScope(scope))
```

Comment:
> chagnes to using(_ = new ForcedScope(scope))
> _ is discard, this is correct syntax i beleive

Yes, `_` is the syntax for C# 7 discards. Happy either way, just didn't want to see an unused local.

Reply:
i removed the (**var** _) as you correctly recommended., thanks for that comment :-)